### ISSUE TITLE
Can't reproduce mAP score on VeRi

### ISSUE THREAD

**Issue by shikhar394** [Xiangyu-CAS/AICity2020-VOC-ReID#27]
Greetings, 

I can't seem to reproduce the results that you've achieved on VeRI using your pre-trained models. 

I ran the script (scripts/veri/test.sh) with the pretrained models from [here](https://drive.google.com/drive/folders/1W8nw3GEYyxZiuDSk_wdXTErHFxtfKfKI). 

However, my mAP is only 37.7% compared to the >80% shown in your log files. Here's my [log](https://pastebin.com/SBjHCPAK). 

Could you please offer some detailed instructions on how to test on VeRi using your pre-trained weights?

Thank you!

**Comment by Xiangyu-CAS** [comment]
Hi, I didn't upload the model pre-trained on VeRi dataset. Based on your log, it seems you were using the model pre-trained on AICity dataset but testing on VeRi dataset. To reproduce accuracy, you need to run "scripts/veri/baseline.sh' and train the model by yourself.  Someone in [ISSUE 26](https://github.com/Xiangyu-CAS/AICity2020-VOC-ReID/issues/26) said he/she had reproduced the resutls on VeRi

**Comment by shikhar394** [comment]
Alright, thanks for sharing that.
